### PR TITLE
Add unittests for mouse module

### PR DESCRIPTION
**Pull request by limingyialpha**

### CODE REVIEW

File: application/Modules/Mouse/Producers/MouseMoveEventProducer.cs [hunk 1]
```diff
@@ -47,7 +47,8 @@ public class MouseMoveEventProducer : DefaultEventQueue<MouseMoveEvent>
         private void GetMousePosition(object stateInfo)
         {
             // get the current mouse position as Point
-            nativeMouse.GetCursorPos(out var currentMousePosition);
+            var currentMousePosition = nativeMouse.GetCursorPos();
+            ///!!!nativeMouse.GetCursorPos(out var currentMousePosition);
```
Comment (Gordin508) [inline application/Modules/Mouse/Producers/MouseMoveEventProducer.cs]: Best remove unneeded lines before merging.

File: application/Modules/Mouse/Producers/MouseMoveEventProducer.cs [hunk 2]
```diff
@@ -73,7 +74,8 @@ private void GetMousePosition(object stateInfo)
         public void StartCapture(INativeMouse nativeM)
         {
             nativeMouse = nativeM;
-            nativeMouse.GetCursorPos(out lastMousePosition);
+            lastMousePosition = nativeMouse.GetCursorPos();
+            ///!!!nativeMouse.GetCursorPos(out lastMousePosition);
```
Comment (Gordin508) [inline application/Modules/Mouse/Producers/MouseMoveEventProducer.cs]: same

**Comment by limingyialpha** [comment]
MouseModuleConfiguration has been added.

**Comment by limingyialpha** [comment]
About the CI:

The failed test is not in the scope of the this module and
is fixed in another PR.

**Pull request merged**